lib/OpenQA/Schema/ResultSet/DeveloperSessions.pm

```diff
@@ -57,7 +59,12 @@ sub register {
     if ($result && !$is_session_already_existing) {
         # hope this IPC call isn't blocking too long (since the livehandler isn't preforking)
         my $client = OpenQA::WebSockets::Client->singleton;
-        $client->send_msg($worker_id, 'developer_session_start', $job_id);
+        try {
+            $client->send_msg($worker_id, 'developer_session_start', $job_id);
+        }
+        catch {
+            log_error("Unable to inform worker about developer session: $_");
```

Comment:
I suggest to not hide changes like this behind a pull request labeled "Refactor". Can you separate this please? If not, please update your PR to tell about more than just "Refactor"

Reply:
It is already a separate commit but I will move it into a separate PR.